### ISSUE TITLE
marimo notebook hangs when using pytorch-ignite metraics

### ISSUE THREAD

**Issue by obsessor-ak1** [marimo-team/marimo#10213]
### Describe the bug

Instantiating a any `ignite.metrics.Metric` instance it just hangs that particular cell executing it. I tried and tested this with `ignite.metrics.Loss` and `ignite.metrics.RunningAverage`. It it just remains hung. It just never terminates, the hourglass keeps spinning...

<img width="1918" height="940" alt="Image" src="https://github.com/user-attachments/assets/494f5add-5b24-411c-be5b-b3c142ba3303" />

Further when you try to stop the cell by using the interrupt button, it doesn't get interrupted, here is the log message when clicking that button:

<img width="1918" height="1078" alt="Image" src="https://github.com/user-attachments/assets/bce56b1f-397f-4e61-b1db-b168f75a3452" />

I also checked for the specific codebases of [PyTorch Ignite](https://github.com/pytorch/ignite), more specifically the source codes of `ignite.metrics.Loss` and `ignite.metrics.RunningAverage` but couldn't find anything problematic there. At least I didn't see any problem in their `__init__` method. Please look into this. Or suggest a fix. The issue is same when running locally as well as on _molab_

### Will you submit a PR?

- [ ] Yes

### Environment

<details>

```
OS: Windows 11
Python: 3.14.0 (Miniconda)
marimo: 0.23.14
```

</details>

### Code to reproduce

```python
# In any marimo code cell run the following
import marimo as mo
import torch
from ignite.metrics import Loss, RunningAverage
loss = Loss(torch.nn.CrossEntropyLoss()) # or any other metric practically
```

**Comment by obsessor-ak1** [comment]
@Light2Dark A huge thanks for fixing the bug....and letting me use my favorite training framework (Ignite) with the power of marimo without sacrificing either of those.....❤️

**Comment by Light2Dark** [comment]
no problem! let us know if you face other issues.
